src/nRF51822.cpp

```diff
@@ -621,10 +624,6 @@ void nRF51822::poll() {
       case BLE_GAP_EVT_SEC_PARAMS_REQUEST:
 #ifdef NRF_51822_DEBUG
         Serial.print(F("Evt Sec Params Request "));
-#if !defined(NRF51_S130) && !defined(S110)
```

Comment:
@sandeepmistry Yes as I can see `timeout` data is no longer part of  in the `ble_gap_sec_params_t` struct in ble_gatt.h definitions for all used softdevices types (s110, s130,s132) and versions (see arduino-nRF5/). So, we can remove that debug printout completly. Do you aggree?